tests/SpecTests/ClientSideEncryptionSpecTest.php

```diff
@@ -95,6 +95,9 @@ public function testClientSideEncryption(stdClass $test, ?array $runOn, array $d
             $this->markTestSkipped($test->skipReason);
         }
 
+        // Checking for mongocryptd availability
+        $this->skipIfMongocryptdIsUnavailableByTest($test);
```

Comment:
The suggested in the source ticket function skipIfClientSideEncryptionIsNotSupported is not good enough for expected check because remote or specific location of mongocryptd can be defined in provided test or in some specific auto encryption options. That's the reason why each test function contains its personal check for mongocryptd.